src/WebJobs.Extensions.DurableTask/EntityQuery.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    /// <summary>
+    /// Query condition for searching the status of entity instances.
+    /// </summary>
+    public class EntityQuery
+    {
+        /// <summary>
+        /// Return entity instances associated with this entity name.
+        /// </summary>
+        public string EntityName { get; set; }
+
+        /// <summary>
+        /// Return entity instances which had operations after this DateTime.
+        /// </summary>
+        public DateTime LastOperationFrom { get; set; } = DateTime.MinValue;
+
+        /// <summary>
+        /// Return entity instances which had operations before this DateTime.
+        /// </summary>
+        public DateTime LastOperationTo { get; set; } = DateTime.MaxValue;
+
+        /// <summary>
+        /// Number of records per one request. The default value is 100.
+        /// </summary>
+        public int PageSize { get; set; } = 100;
+
+        /// <summary>
+        /// ContinuationToken of the pager.
+        /// </summary>
+        public string ContinuationToken { get; set; }
+
+        /// <summary>
+        /// Determines whether the query will include the input of the entity.
```

Comment:
Should be "_state_ of the entity" rather than "_input_ of the entity" since entities don't have a concept of "inputs".